WPFHospitalEditor/Service/MapObjectService.cs

```diff
@@ -38,5 +46,41 @@ private List<MapObject> colorSelectedObject(List<MapObject> allMapObjects)
             }
             return allMapObjects;
         }
+
+        public List<MapObject> checkMapObjectSearchInput(List<MapObject> mapObjects, string textBoxInput, string comboBoxTextInput)
+        {
+            List<MapObject> searchResult = new List<MapObject>();
+            MessageBox.Show("Lista: " + mapObjects.Count + " TextBox: " + textBoxInput + "ComboBox: " + comboBoxTextInput);
+            foreach (MapObject mapObject in mapObjects)
```

Comment:
MessageBox.Show() shouldn't be in code? This repeats after foreach.

Reply:
I left that by accident. I was testing if method gets right values and forgot to delete it.